onadata/apps/logger/tests/data_migration/test_backup.py

```diff
@@ -1,40 +1,23 @@
-from datetime import datetime
-
-from onadata.apps.logger.models import Instance, XForm, BackupInstance, BackupXForm
+from onadata.apps.logger.models import Instance, BackupInstance, BackupXForm
 from onadata.apps.logger.data_migration.backup_data import (
     create_xform_backup, backup_xform, backup_survey
 )
-from onadata.apps.logger.data_migration.restore_backup import (
-    copy_attrs, BackupRestorer, BackupRestoreError
-)
-
-from .common import MigrationTestCase, SecondMigrationTestCase
-from . import fixtures
+from .common import MigrationTestCase
 
 
 class BackupSurveysTests(MigrationTestCase):
     def test_backup_xform(self):
         BackupXForm.objects.all().delete()
         xform_backup = backup_xform(self.xform)
-        sorting_key = lambda x: x.title
-        self.assertEqual(xform_backup.xml, self.xform.xml)
+        self.assertEqual(self.xform.xml, xform_backup.xml)
```

Comment:
maybe this could be a part of assertEqual below, instead of getting a separate one